google-api-client/src/main/java/com/google/api/client/googleapis/json/GoogleJsonResponseException.java

```diff
@@ -141,6 +141,9 @@ public static GoogleJsonResponseException from(JsonFactory jsonFactory, HttpResp
     }
     // message
     StringBuilder message = HttpResponseException.computeMessageBuffer(response);
+    (message.length() == 0 ? message : message.append(StringUtils.LINE_SEPARATOR))
+        .append("Request URL: ")
```

Comment:
Would it better to just append an URL on a new line without `Request URL: ` prefix?

Reply:
@elharo what do you think about this? Should we include `Request URL:` prefix in the error message?

```
500 Server error
http://storage.googleapis.com/bucket/...
```

vs

```
500 Server error
Request URL: http://storage.googleapis.com/bucket/...
```

Reply:
I don't have a strong opinion here. Maybe include it? Either way is fine with me.

Reply:
Ack, will leave it as is.